transformer_engine/jax/csrc/utils.h

```diff
@@ -93,6 +96,27 @@ class cudaDevicePropertiesManager {
     cudaDeviceProp prop_;
 };
 
+class FusedAttnOffsetManager {
```

Comment:
I don't expect it to cause problems, but it does seem strange to have a global offset value when each attention module may have a different seed.